privacyidea/static/components/token/controllers/tokenControllers.js

```diff
@@ -667,18 +667,29 @@ myApp.controller("tokenEnrollController", ["$scope", "TokenFactory", "$timeout",
 
 myApp.controller("tokenImportController", ['$scope', 'Upload', 'AuthFactory',
                                            'tokenUrl', 'ConfigFactory', 'inform',
+                                           'gettextCatalog',
                                            function ($scope, Upload,
                                                      AuthFactory, tokenUrl,
-                                                     ConfigFactory, inform) {
+                                                     ConfigFactory, inform,
+                                                     gettextCatalog) {
     $scope.formInit = {
         fileTypes: ["aladdin-xml", "OATH CSV", "Yubikey CSV", "pskc"]
     };
+
+    $scope.verify_pskc_opts = {
+        no_check: gettextCatalog.getString('Do not verify the authenticity'),
+        check_fail_soft: gettextCatalog.getString('Skip tokens that can not be verified'),
+        check_fail_hard: gettextCatalog.getString('Abort operation on unverifiable token'),
+    }
+
     // These are values that are also sent to the backend!
     $scope.form = {
         type: "OATH CSV",
         realm: ""
     };
 
+
```

Comment:
please remove these newlines

Reply:
fixed in 32f27b19081a17294986230368447e909ad3762e.